sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -378,6 +379,23 @@ def run_Flatten(self, transform_node):
           PropertyNames.ENCODING: step.encoding,
           PropertyNames.OUTPUT_NAME: PropertyNames.OUT}])
 
+  def apply_WriteToBigQuery(self, transform, pcoll):
```

Comment:
I'm a bit confused. This is the dataflow_runner.py file so we're only overriding the one in dataflow runner. We have always used the streaming api for the pipelines in the direct runner.